lib/PlacesProvider.js

```diff
@@ -194,17 +194,25 @@ Links.prototype = {
       limit = HISTORY_RESULTS_LIMIT;
     }
 
-    let sqlQuery =  `SELECT moz_places.url as url, moz_favicons.data as favicon, mime_type as mimeType,
-                            title, frecency, last_visit_date as lastVisitDate,  "history"  as type
+    let sqlQuery =  `SELECT moz_places.url as url,
+                            moz_favicons.data as favicon,
+                            mime_type as mimeType,
+                            moz_places.title,
+                            frecency,
```

Comment:
nit: inconsistent use of column declaration

consider doing this instead:
```sql
SELECT moz_places.url as url,
               moz_favicons.data as favicon,
               moz_favicons.mime_type as mimeType,
               moz_places.title,
               moz_places.frecency,
               moz_places.last_visit_date as lastVisitDate,
               moz_bookmarks.guid as bookmarkGUID,
               "history"  as type
FROM moz_places
LEFT JOIN moz_favicons
ON moz_places.favicon_id = moz_favicons.id
LEFT JOIN moz_bookmarks
ON moz_places.id = moz_bookmarks.fk
WHERE hidden = 0 AND last_visit_date NOTNULL
ORDER BY lastVisitDate DESC, frecency DESC, url
LIMIT :limit;
```

The query becomes much more readable.